canonical-web-and-design/ubuntu.com
# Add Spanish meta image and fix URL

## Done

- Replace meta image with Spanish title
- Remove accents from file path

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/guia-implementacion-openstack
- Check meta image is correct 
- Check URL is clean


## Issue / Card

Fixes [#8366](https://github.com/canonical-web-and-design/ubuntu.com/issues/8366)
